android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/ActivityResultSender.kt

```diff
@@ -1,7 +1,35 @@
 package com.solana.mobilewalletadapter.clientlib
 
 import android.content.Intent
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.annotation.GuardedBy
 
-interface ActivityResultSender {
-    fun launch(intent: Intent)
+class ActivityResultSender(
```

Comment:
Can we also make this final? I don't see a reason for consumers to extend/inherit this

Reply:
Very good point

Reply:
It is, make sure you're looking at latest code @ankur2136

Reply:
I'm trying to make the class final, and Android studio is saying it's redundant

Reply:
kotlin classes are final by default